prql-compiler/tests/integration/main.rs

```diff
@@ -25,15 +25,33 @@ mod tests {
             }
 
             // compile
-            let sql = prql_compiler::compile(&prql, None).unwrap();
+            let opts = prql_compiler::sql::Options::default()
+                .with_dialect(prql_compiler::sql::Dialect::SQLite)
+                .no_format()
+                .some();
+            let sql = prql_compiler::compile(&prql, opts).unwrap();
+            let sqlite_out = sqlite::query_csv(&sqlite_conn, &sql);
 
             // save both csv files as same snapshot
-            assert_snapshot!(sqlite::query_csv(&sqlite_conn, &sql));
-            assert_snapshot!(duckdb::query_csv(&duckdb_conn, &sql));
+            let opts = prql_compiler::sql::Options::default()
+                .with_dialect(prql_compiler::sql::Dialect::DuckDb)
+                .no_format()
+                .some();
+            let sql = prql_compiler::compile(&prql, opts).unwrap();
+            let duckdb_out = duckdb::query_csv(&duckdb_conn, &sql);
+            assert_eq!(sqlite_out, duckdb_out);
 
             if let Some(pg_client) = &mut pg_client {
-                assert_snapshot!(postgres::query_csv(pg_client, &sql));
+                let opts = prql_compiler::sql::Options::default()
+                    .with_dialect(prql_compiler::sql::Dialect::PostgreSql)
+                    .no_format()
+                    .some();
+                let sql = prql_compiler::compile(&prql, opts).unwrap();
+                let pg_out = postgres::query_csv(pg_client, &sql);
+                assert_eq!(sqlite_out, pg_out);
             }
+
+            assert_snapshot!(sqlite_out);
```

Comment:
Very nitty comment: to the extent we want the source expression in the insta snapshot, we can pass it with the three argument form.

Reply:
I tried that, but the macro understood first argument as snapshot name.